### ISSUE TITLE
TOC focus state not clearing after new active state selected

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/ibm-dotcom-library#1633]
## Detailed description

> Describe in detail the issue you're having.

The Table of Contents is not clearing the `focus` state after a new `active` state is selected.   When the blue bar is on the `focus` state, the blue bar should clear.

![Screen Shot 2020-03-09 at 3.11.42 PM.png](https://images.zenhubusercontent.com/5cacfd2d41f23a1e95e068dc/9ec52a9d-3035-490c-894a-bda45a45ae49)

> Is this a feature request (new component, new icon), a bug, or a general
> issue?

Bug

> Is this issue related to a specific component?

Table of contents

> What did you expect to happen? What happened instead? What would you like to
> see changed?

The focus state should clear once a new active state is selected.

> What browser are you working in?

Mac > Chrome

> What version of the IBM.com Library are you using?

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

Web simplification

## Steps to reproduce the issue

1. [Learn page](http://ibmdotcom-nextjs-test.mybluemix.net/learn)
2. Click any of the Table of contents options that are not the first item
3. Scroll to another section
4. See focus state **and** active state on two different items

## Additional information

- Screenshots or code
- Notes

**Comment by RobertaJHahn** [comment]
@Praveen-sr @chetankan Can you have Rahm take a look at this to see whether this is an accessibility issue?

**Comment by RobertaJHahn** [comment]
3/10/2020;  Rahm said that it is working as expected.  Closing this issue.

**Comment by Praveen-sr** [comment]
@RobertaJHahn Checked this with the accessibility team, focus state is clearing after new active state Clicked / selected.
